migrations/V94__Update_General_Characteristics_AMSModelID_Abbreviation_DemoCode.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE plan_general_characteristics
+  ADD COLUMN demo_code zero_string,
+  ADD COLUMN ams_model_id uuid,
+  ADD COLUMN model_abbreviation zero_string;
```

Comment:
@OddTomBrooks , did you think about including `model_abbreviation` on `model_plan`?  I think that it could make more sense there in terms of data organization (since it's an abbreviation of model name)

Reply:
I did think about it. Rationally it made sense to me for it to go on either general characteristics or model plan. Since I didn't have a strong feeling either way, I simply added it to general characteristics as it incurred fewer file changes. If you feel strongly about this I can migrate this field to model plan.

Reply:
Thanks @OddTomBrooks , I think I prefer it in `model_plan`, but  `general_characterstics` could be ok. If you and @ClayBenson94 prefer general characteristics, that's ok. Thoughts?